package.json

```diff
@@ -11,7 +11,8 @@
     "dependencies": {
         "electron": "1.8.4",
```

Comment:
Please update to the latest project linting definitions (all other linting dependencies can be removed except for eslint-config-fluid) as seen at https://github.com/fluid-project/infusion/blob/master/package.json#L43 and https://github.com/GPII/universal/blob/master/Gruntfile.js#L18

Reply:
Please also do a general sweep of outdated dependencies via npm outdated

Reply:
We changed the linting definitions and cleaned up the newly emerged lint warnings. Most of the dependencies listed in the `package.json` file are related either to the build or testing process and we are not sure whether we are the right people to make these changes. Perhaps @javihernandez and @the-t-in-rtf can help?

Reply:
@georgitodorov, IMO we should all be comfortable routinely updating the dependencies based on the output of `npm outdated`.  There may be cases where we cannot upgrade beyond a particular point, but in my experience it's better to update often and flag any problems.  Among other things, we pick up security and bug fixes as part of our ongoing work rather than having to charge someone with monitoring and performing the work separately.

Reply:
Okay, I updated the references to the testing-related modules, did some preliminary checks and I think that the testing infrastructure is working fine. As for the `electron-packager`, I see that the module's major version has been bumped by 4 and I am apprehensive to update it without knowing how to check if it is working. That is why my suggestion is to wait for @javihernandez to come back and verify if everything will be correct. As for the update of Electron, we have a separate task for doing this (https://issues.gpii.net/browse/GPII-3044) which may require more work (e.g. more testing to discover if everything behaves correctly). That is why I would propose to leave the update of Electron for another time.